utils/es_context.py

```
@@ -0,0 +1,17 @@
+    if not ctx:
+        return
+    try:
+        if hasattr(ctx, 'request_context') and ctx.request_context:
+            request = ctx.request_context.request
+            if request and hasattr(request, 'headers'):
+                headers_dict = dict(request.headers)
+                es_config = get_es_config_from_headers(headers_dict)
+                if es_config:
+                    current_es_config.set(es_config)
+    except Exception:
+        pass
```

Ruff (0.16.3)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)